<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlocksPerSecondDialog</class>
 <widget class="QDialog" name="BlocksPerSecondDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blocks per second calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/appicon/appicon.qrc">
    <normaloff>:/appicon/16.png</normaloff>:/appicon/16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Move time:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinBlocks">
     <property name="suffix">
      <string> blocks</string>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Size of one block:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toggleSign">
     <property name="text">
      <string notr="true">+/-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBlockSize"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="spinSecond">
     <property name="prefix">
      <string>per </string>
     </property>
     <property name="suffix">
      <string> second</string>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/appicon/appicon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BlocksPerSecondDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BlocksPerSecondDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
